### PR TITLE
Added grid component and docs

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**
created different variations of grid component along with docs. Please review

### CODE REVIEW

File: components/Grid/grid.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+@import "/docs.css";
+
+
+
+.two-item-container{
+    display: grid;
+    grid-template-columns: repeat(2,1fr);
+    padding: 1rem;
+    gap:20px;
```
Comment (bmendhe23) [inline components/Grid/grid.css]: You can use rem here.

**Review by bmendhe23 — APPROVED** [review]
Looks good to me. I have suggested a change you can do that, otherwise it is good to go.

File: components/Grid/grid.css (hunk 1 above)
Comment (muskan-gupta-0803) [reply components/Grid/grid.css]: okay

**Comment by muskan-gupta-0803** [comment]
> Looks good to me. I have suggested a change you can do that, otherwise it is good to go.

thanks for the review

**Review by PrashanthreddyAinala — COMMENTED** [review]
Everything looks good for me

**Pull request merged**